<launch>
  <node pkg="ros_traj_gen_utils" type = "traj_exe" 
    name = "traj_exe" output = "screen"  clear_params="true" >
     <rosparam command="load" file="$(find ros_traj_gen_utils)/config/std_launch.yaml"/>
  </node>
</launch>
